### PR TITLE
LPS-195331 Add configuration button to Display Page 

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-195331)

## Steps

- Go to Design > Page Templates > Display Page Templates > create a DPT
Screenshot 2023-09-05 at 13 23 39

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/servlet/taglib/util/DisplayPageActionDropdownItemsProvider.java [hunk 1]
```diff
@@ -161,6 +167,36 @@ hasUpdatePermission && _isShowDiscardDraftAction(),
...
+			"/layout_admin/edit_layout"
+		).setRedirect(
+			currentURL
+		).setBackURL(
+			currentURL
+		).setParameter(
+			"groupId", _layoutPageTemplateEntry.getGroupId()
+		).setParameter(
+			"selPlid", _draftLayout.getPlid()
+		).buildPortletURL(
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/servlet/taglib/util/DisplayPageActionDropdownItemsProvider.java:189]: @DiegoHu97 you can use buildString directly

File: modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/servlet/taglib/util/DisplayPageActionDropdownItemsProvider.java [hunk 2]
```diff
@@ -161,6 +167,36 @@ hasUpdatePermission && _isShowDiscardDraftAction(),
 		).build();
 	}
 
+	private UnsafeConsumer<DropdownItem, Exception>
+		_getConfigureDisplayPageEntryActionUnsafeConsumer() {
+
+		String currentURL = PortalUtil.getCurrentURL(_httpServletRequest);
+
+		String portletURLBuilder = PortletURLBuilder.create(
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/servlet/taglib/util/DisplayPageActionDropdownItemsProvider.java:175]: @DiegoHu97 Could you please add a better name?

**Comment by DiegoHu97** [comment]
@ealonso Updated! 😄

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**